sorting/comb_sort.cpp

```diff
@@ -14,7 +14,7 @@ int FindNextGap(int x) {
     return std::max(1, x);
 }
 
-void CombSort(int a[], int l, int r) {
+void CombSort(int b[], int l, int r) {
```

Comment:
instances of `a[]` type inputs should be replaced with pointers `*a`

Reply:
`*a[]` or `*a`?

Reply:
I'd suggest, once #888  is merged to master, rebase your branch on the master. You'll get merge conflicts during which you can undo your changes to comb_sort.cpp.
or else you can close this and create a new pull. 

It is a good practise to commit one file changes at a time.